### PR TITLE
Extract stripe keys in config

### PR DESCRIPTION
**Pull request by jclariviere**

### CODE REVIEW

File: server/src/main/resources/com/arcbees/beeshop/server/config.properties.example [hunk 1]
```diff
@@ -0,0 +1 @@
+stripePrivateKey = sk_<your-private-key>
```
Comment (christiangoudreau) [inline server/src/main/resources/com/arcbees/beeshop/server/config.properties.example:1]: yeah. shouldn't be shared... :D

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,6 @@ target/
 extras/
 /gen
 .DS_Store
+
+client/src/main/resources/com/arcbees/beeshop/client/Config.properties
+server/src/main/resources/com/arcbees/beeshop/server/config.properties
```
Comment (Chris-V) [inline .gitignore:9]: What is thrown / displayed if you don't create the properties files?

**Comment by Chris-V** [comment]
LGTM. Maybe a more convenient way to set the configs would be nice (maven filtering, environment variables, etc)

File: .gitignore (hunk 2 above)
Comment (jclariviere) [inline .gitignore:9]: iirc, client doesn't build and server throws an exception on startup.

**Comment by Pacane** [comment]
LGTM

**Pull request merged**